.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--
+  For work in progress use the GitHub draft pull request feature.
+-->
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+- [ ] this PR contains breaking changes!
+
+# Resolves / is related to
+
+<!-- Relates to #IssueNumber1, #IssueNumber2 -->
+<!-- Closes #IssueNumber3 -->
+<!-- Closes #IssueNumber4 -->
+
+# What is affected by this PR
+
+- [ ] OpenFaaS Function Exchange Format
+- [ ] Message Format
+- [ ] Documentation
+
+# Checklist
+
+- [ ] Ensure that new source files include the [license header](https://github.com/UST-MICO/mico/blob/master/CONTRIBUTING.md#source-file-headers)
+- [ ] Ensure [JavaDoc is up to date](https://github.com/UST-MICO/mico/tree/master/docs#build-the-documentation-locally)
```

Comment:
This is only relevant for mico core and is not setup in this repo.

Reply:
The license header should be included. Maybe we should add an issue to setup the JavaDoc part for this repo in the future.